src/js/request_scheduler.js

```diff
@@ -234,7 +243,7 @@ class RequestScheduler {
             'running' : this.running_count,
             'completed' : this.completed_count,
             'errors' : this.error_count,
-            'cache_hits' : this.cache.hitCount(),
```

Comment:
not asking for this to change btw - trailing commas are almost as common a subject for code review wars as tabs vs spaces. Given there is no coding standard for this project yet, you should probably not waste your time fixing stuff like this.
btw: spaces not tabs! :)